apps/earthbound/components/MembershipTimeline.tsx

```
@@ -97,7 +97,7 @@ export default function MembershipTimeline() {
               >
                 {step.position === 'left' ? (
                   <>
-                    <div className="pr-8 py-2">{step.title}</div>
+                    <div className="pr-8 py-2" dangerouslySetInnerHTML={{ __html: step.title }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)